openmetadata-service/src/main/java/org/openmetadata/service/apps/bundles/insights/DataInsightsReportApp.java

```diff
@@ -294,6 +294,7 @@ private DataInsightDescriptionAndOwnerTemplate createDescriptionTemplate(
           PERCENTAGE_OF_ENTITIES_WITH_DESCRIPTION_BY_TYPE,
           currentPercentCompleted,
           currentPercentCompleted - previousPercentCompleted,
+          (int) currentCompletedDescription,
```

Comment:
should this be a long?

Reply:
nope it is count of entities with description